assets/src/domain/eventEditor/ui/datetimes/datesList/views/List/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
This file should goto `/ui/datetimes/datesList/tableView` and file should be `TableView.tsx`

Reply:
Updated: https://github.com/eventespresso/event-espresso-core/pull/2431/commits/85382e6282f4968ffa084ad4219a6e5f6660316d